README.md

```diff
@@ -3,3 +3,17 @@
 # rush
 
 rust implementation of Aleph protocol
+
+# examples
+
+There is a dummy implementation of an honest committee member that is not
+cryptographically secure and servers only as a working example of what has to be
+implemented and not how it should be implemented. For detailed information on
+what is required from a client using this library see [Damian's description](??).
```

Comment:
While I very strongly approve of giving credit I think this should probably be referred to under a different name. :>

Reply:
Placeholder until Damian's PR is not merged :)